### PR TITLE
Added VSCode project's config

### LINKED ISSUE #42: Add linting for front-end part

### PR DESCRIPTION
**Pull request by t20100**
This PR adds configuration of VSCode for lint on save.
The web part is copied from `h5web` (what else?).

See commit https://github.com/silx-kit/jupyterhub_moss/pull/43/commits/aada2d8ca745f57060003840c1448b0c05a36811 for the actual change.
The remaining is applying those rules to existing code and md.

related to #42

### CODE REVIEW

**Review by loichuder — APPROVED** [review]
Nice initiative :+1:

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```
[the remark this answers is not in the document]
Comment (t20100) [reply .editorconfig:9]:
```suggestion
insert_final_newline = true

```

**Comment by t20100** [comment]
There's an issue with `indent`, the usual Python indent being 4...

**Comment by axelboc** [comment]
You can probably add an override like for Markdown but with the option `tabWidth`.

**Review by axelboc — APPROVED** [review]

**Comment by t20100** [comment]
Indeed, that was easy to fix.

**Pull request merged**